src/ml_tooling/storage/file.py

```diff
@@ -0,0 +1,106 @@
+from ml_tooling.storage.base import Storage, StorageEnvironment
+from ml_tooling.utils import MLToolingError
+
+from typing import List, Any
+from contextlib import contextmanager
+from pathlib import Path
+from joblib import load, dump
+
+
+class FileStorage(Storage):
+    
+    def __init__(self, dir_path=None):
+        self.dir_path = Path.cwd() if dir_path is None else Path(dir_path)
+        self.filename = None
+        self.file = None
```

Comment:
These are not used. 

Is the main idea of the class that the FileStorage is given the path of the folder and then you only give the file name to save and load? If this is the case should load and save take a file name not a path

Reply:
true, self.file isnt used.
self.filename is used, the baseclass sets the filename when saving its estimator. you can use it to set the filename manually if you want to use the FileStorage directly

Reply:
Okay. I think it would be more intuitive to give the filename directly to the .save method instead of being a "hidden" attribute

Reply:
ok, sounds good to me too